s3.go

```diff
@@ -56,6 +56,22 @@ type Config struct {
 	RootDirectory       string
 	Workers             int
 	CredentialsEndpoint string
+	KeyTransform        string
+}
+
+var KeyTransforms = map[string]func(ds.Key) string{
+	"default": func(k ds.Key) string {
```

Comment:
Calling this "default" isn't very descriptive, how about "identity" or something like that?